docs/self-hosted-appcircle/install-server/helm-chart/installation/openshift.md

```diff
@@ -0,0 +1,535 @@
+    organizationName: spacetech
+    # Initial admin user email for Appcircle server
+    initialUsername: "admin@example.com"
+    # Initial admin password - Should contain: min 6 chars, 1 lowercase, 1 uppercase, 1 number
+    # You can create a secret with the password or directly enter the password here
+    initialPassword: "superSecretAppcirclePassword1234"
+    image:
+      # Appcircle keycloak image repository path
+      repository: europe-west1-docker.pkg.dev/appcircle/docker-registry/appcircle-keycloak
+
+# Internal Ingress controller configuration
+ingress-nginx:
+  enabled: false
+
+# Appcircle vault configuration
+vault:
+  server:
+    image:
+      # Appcircle vault image repository path
+      repository: europe-west1-docker.pkg.dev/appcircle/docker-registry/appcircle-vault
+
+# Web event Redis configuration
+webeventredis:
+  # Enable TLS for Redis connections
+  tls:
+    enabled: true
+  # Ingress configuration for Redis
+  ingress:
+    enabled: true
+    tls: true
+```
+
+</details>
+
+  </TabItem>
+</Tabs>
+
+### 2. Remove Sensitive Information From `values.yaml`
+
+**Remove sensitive information** such as Appcircle initial user password, SMTP password, SSL certificates, and other secrets from the `values.yaml` **for production environments**, by checking the [Sensitive Values](/self-hosted-appcircle/install-server/helm-chart/configuration/sensitive-configuration) documentation.
+
+### 3. Add the Appcircle Helm Repository
+
+**Add the Appcircle Helm repository** to the configuration of Helm:
+
+```bash
+helm repo add appcircle https://helm-package.appcircle.io && \
+helm repo update
+```
+
+### 4. Install the Appcircle Server
+
+**Run the following Helm command** to install the Appcircle server chart.
+
+In this example, we deploy the Appcircle server to a single namespace, using **`appcircle`** as the **namespace** and **`appcircle-server`** as the Helm **release name**.
+
+```bash
+helm install appcircle-server appcircle/appcircle \
+  --timeout 1200s \
+  -n appcircle \
+  -f values.yaml
+```
+
+:::warning
+If you need or want to change the release name, please note that it should be 18 characters or fewer.
+:::
+
+You can watch the Appcircle server installation with any OpenShift monitoring tool. The installation process duration depends on factors such as network speed and the processing power of your Kubernetes nodes. Typically, the installation may take up to **10 to 15 minutes**.
+
+To make sure that the Appcircle server is installed successfully, you can run the command below and wait to finish:
+
+```bash
+kubectl wait --for=condition=ready pod \
+  -l app.kubernetes.io/instance=appcircle-server \
+  -n appcircle --timeout 1200s && \
+  echo "Appcircle is ready to use. Happy building! "
+```
+
+## Post-installation Steps
+
+### 1. Add DNS Records
+
+List the services with `kubectl` to check the IP address of the OpenShift default router
+
+```bash
+kubectl get svc -n openshift-ingress
```

Comment:
AS A GENERAL COMMENT, all `kubectl` commands in this document should be replaced with the equivalent `oc` command in order to talk in OpenShift language or domain.